reporter/otlp_reporter.go

```diff
@@ -59,8 +62,42 @@ type OTLPReporter struct {
 	pkgGRPCOperationTimeout time.Duration
 }
 
+// TODO: Remove before merging, only used for testing
```

Comment:
The change looks good to me. Just removing this testing part and I'm happy to approve.

Reply:
Done